recipe/build.sh

```diff
@@ -6,11 +6,32 @@
 # but it works fine to get our first build.
 unset LDFLAGS
 
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+#
+# Also, included a workaround so that `-stdlib=c++` doesn't go to
+# `gfortran` and cause problems.
+#
+# https://github.com/conda-forge/toolchain-feedstock/pull/8
+OLD_LDFLAGS="${LDFLAGS}"
```

Comment:
Is this some kind of future-proofing? ``LDFLAGS`` is unset above, so this isn't doing anything now